pkg/debugger/proxy.go

```diff
@@ -20,8 +20,8 @@ func (ds DebugConfig) getProxies() http.Handler {
 	// the type (map) required by the printProxies function.
 	listConnected := func() map[certificate.CommonName]time.Time {
 		proxies := make(map[certificate.CommonName]time.Time)
-		for cn, proxy := range ds.proxyRegistry.ListConnectedProxies() {
-			proxies[cn] = (*proxy).GetConnectedAt()
+		for _, proxy := range ds.proxyRegistry.ListConnectedProxies() {
+			proxies[proxy.GetCertificateCommonName()] = (*proxy).GetConnectedAt()
```

Comment:
Should we be calling `GetName()` instead of `GetCertificateCommonName()`?
```suggestion
			proxies[proxy.GetName()] = (*proxy).GetConnectedAt()
```

Reply:
on first glance you would think so, but this is actually creating a new map from the old map, keyed by common name. I have some future changes that are removing the GetCommonName method, so that will change in the future, and we can just reuse the existing map.

Reply:
Thank you for clarifying!